### PR TITLE
Render jobs from api

### PR DESCRIPTION
**Pull request by ksk5280**
Now hits the production endpoint!

Awaiting changes on the endpoint to display description and technologies properly.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -9,14 +9,19 @@ require('../public/custom.css');
...
+    $.getJSON('https://lookingforme.herokuapp.com/api/v1/jobs', (response) => { this.setState({ jobs: response }) });
   }
 
   render() {
     return (
       <div className="application">
           <Header />
         <div className="container">
-          <JobListings />
+          <JobListings jobs={this.state.jobs.jobs}/>
```
Comment (ksk5280) [inline lib/index.js]: The endpoint is going to be changed so that we will be able to call 'this.state.jobs' instead of 'this.state.jobs.jobs'

Comment (brianrip) [inline lib/index.js]: @nickybobby This is why we need to get rid of the `job` key on the JSON job object.

File: lib/index.js [hunk 2]
```diff
@@ -70,22 +75,20 @@ class JobSeachBar extends React.Component {
...
-  componentDidMount() {
-    let sampleJobs = require('./sample-jobs');
-    this.setState({ jobs: sampleJobs });
   }
 
   render() {
-    return(
-      <div>
-        { this.state.jobs.map((job) => {
+    if (this.props.jobs !== undefined) {
```
Comment (ksk5280) [inline lib/index.js]: Is there a way to get around using the if-else statement or is this common practice? When the page is initially rendered 'this.props.jobs' is undefined so on line 84 'this.props.jobs.map' results in an error.

@rrgayhart @applegrain

File: lib/index.js [hunk 3]
```diff
@@ -70,28 +75,18 @@ class JobSeachBar extends React.Component {
...
-    // let sampleJobs = require('./sample-jobs');
-    // this.setState({ jobs: sampleJobs });
-    $.getJSON('https://lookingforme.herokuapp.com/api/v1/jobs', (response) => { this.setState({ jobs: response }) });
-    console.log('response', this.state)
   }
 
   render() {
-    console.log('render', this.state.jobs.jobs, 'props', this.props)
-    if (this.state.jobs.jobs !== undefined) {
+    if (this.props.jobs !== undefined) {
```
Comment (brianrip) [inline lib/index.js]: @rrgayhart Perhaps we will learn a work around on this from the async lesson?

File: lib/index.js (hunk 2 above)
Comment (cheljoh) [inline lib/index.js]: I agree, we had this problem yesterday when rendering jobs from the file. I am having similar issues when rendering job shows. Let's keep this for now and refactor it later! Create and issue and someone can research ways to do this. 

@ksk5280 @brianrip

**Comment by cheljoh** [comment]
Nice job Kimi! We will work on refactoring the if/else and the jobs.jobs next week (or today if there is time).

**Comment by brianrip** [comment]
Great to have the real data. Some refactoring needed on the unique keys on jobs and job.technologies. Nick will be submitting a PR soon to allow those refactors.

**Pull request merged**